templates/rust/src/main.rs

```diff
@@ -17,7 +17,7 @@ extern crate tiny_http;
 use clap::{App, Arg};
 
 #[cfg(not(feature = "dev"))]
-use std::thread;
+/// use std::thread;
```

Comment:
if it works on build and dev, remove both lines

Reply:
i commented that out because it was throwing a warning sometimes